server/query.js

```diff
@@ -25,10 +25,10 @@ populate_queries = [ // see populate_tables.sql
     `COPY Course (name) FROM '${process.env.CSV_PATH}course.csv' DELIMITER ',' CSV HEADER`,
     `COPY Teaches (course_id, staff_id) FROM '${process.env.CSV_PATH}teaches.csv' DELIMITER ',' CSV HEADER`,
     `COPY EnrolledIn (course_id, student_id) FROM '${process.env.CSV_PATH}enrolledin.csv' DELIMITER ',' CSV HEADER`,
-    `COPY Assignment (course_id, name, deadline, max_grade) FROM '${process.env.CSV_PATH}assignment.csv' DELIMITER ',' CSV HEADER`,
-    `COPY Question (course_id, assignment_name, number, max_grade, description) FROM '${process.env.CSV_PATH}question.csv' DELIMITER ',' CSV HEADER`,
-    `COPY QuestionSubmission (student_id, course_id, assignment_name, question_number, file_path, grade, staff_comments) FROM '${process.env.CSV_PATH}question_submission.csv' DELIMITER ',' CSV HEADER`,
-    `COPY AssignmentSubmission (student_id, course_id, assignment_name, grade, is_submitted) FROM '${process.env.CSV_PATH}assignment_submission.csv' DELIMITER ',' CSV HEADER`    
+    `COPY Assignment (course_id, name, deadline, max_grade, description) FROM '${process.env.CSV_PATH}assignment.csv' DELIMITER ',' CSV HEADER`,
```

Comment:
Our production data doesn't seem to currently have description, but I didn't seem to run into any issues importing the data (production is just NULL).

Reply:
LTGM tho!

Reply:
max added descriptions to the prod data so it should be good now